liberty-maven-plugin/src/it/kernel-install-feature-test/install-usr-feature-ext-it/src/test/java/net/wasdev/wlp/test/feature/it/InstallUsrFeatureToExtTest.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * (c) Copyright IBM Corporation 2018.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package net.wasdev.wlp.test.feature.it;
+
+import static junit.framework.Assert.*;
+import org.junit.Test;
+import org.junit.AfterClass;
+import java.io.File;
+import java.io.IOException;
+import org.apache.commons.io.FileUtils;
+import java.io.FilenameFilter;
+
+public class InstallUsrFeatureToExtTest {
+	
+	static File mavenLocalRepo = new File(System.getProperty("user.home")+ "/.m2/repository");
+	static File userTestRepo = new File(mavenLocalRepo, "test/user/test/features");
+	
+	public static boolean deleteFolder(final File directory) {
+		if (directory.isDirectory()) {
+			File[] files = directory.listFiles();
+			if (null != files) {
+				for (File file : files) {
+					if (file.isDirectory()) {
+						deleteFolder(file);
+					} else {
+						if (!file.delete()) {
+								file.deleteOnExit();
+						}
+					}
+				}
+			}
+		}
+		if(!directory.delete()){
+			directory.deleteOnExit();
+			return false;
+		}
+		return true;
+	}
+	
+    
+    @Test
+    public void testUsrFeatureExtInstall() throws Exception {
+    	try {
+			File featureFile = new File("liberty/wlp/usr/cik/extensions/testExt/lib/features/testesa1.mf");
```

Comment:
Is this a valid path? Wondering about the `/cik/` portion. I did not see that in the pom.xml.

Reply:
It is an arbitrary extension path set in testExt.properties in the /resource directory.